src/main/groovy/edu/oregonstate/mist/locations/frontend/db/LocationDAO.groovy

```diff
@@ -46,7 +46,7 @@ class LocationDAO {
      */
     String getById(String id) {
         try {
-            return "${ESFullUrl}/${id}/_source".toURL().text
+            return "${ESFullUrl}/${id?.toLowerCase()}/_source".toURL().text
```

Comment:
didn't know that. sorta like jquery. it might be worth adding a comment to make this clear to future maintainers.